src/client/cli/cmd/set.cpp

```diff
@@ -116,14 +116,16 @@ mp::ParseCode cmd::Set::parse_args(mp::ArgParser* parser)
     if (status == ParseCode::Ok)
     {
         const auto args = parser->positionalArguments();
+
         if (args.size() != 1)
         {
             cerr << "Need exactly one key-value pair (in <key>=<value> form).\n";
             status = ParseCode::CommandLineError;
         }
         else
         {
-            const auto keyval = args.at(0).split('=', QString::KeepEmptyParts);
+            const auto keyval = args.at(0).split('=', Qt::KeepEmptyParts);
+
```

Comment:
It appears you added some stray newlines (in an otherwise empty commit).

Reply:
Cleaned up. The only intended change here was to silence a deprecated namespace warning.

Reply:
The problem is that doesn't work with earlier Qt versions:

```
./src/client/cli/cmd/set.cpp:126:59: error: ‘KeepEmptyParts’ is not a member of ‘Qt’
  126 |             const auto keyval = args.at(0).split('=', Qt::KeepEmptyParts);
      |                                                           ^~~~~~~~~~~~~~
```

So we decided to live with these warnings for now.